BoxContentSDK/BoxContentSDK/Operations/BOXAPIMultipartToJSONOperation.m

```diff
@@ -169,9 +191,8 @@ - (NSDictionary *)HTTPHeaders
 
 - (void)abortWithError:(NSError *)error
 {
-    [self.connection cancel];
+    self.error = error;
```

Comment:
The callback would get called from canceling the task. If this is so early in the process that a task does not yet exist (which can happen on initial stream setup), then setting the error is sufficient because it gets checked in that setup process.